applications/dashboard/models/class.usermodel.php

```diff
@@ -2656,16 +2657,23 @@ private function addIpFilters($ip, $fields = []) {
         $sql->reset();
 
         // Get all users that matches the IP address.
+        $userIDs = [];
+
         $sql
             ->select('UserID')
             ->from('UserIP')
             ->where('IPAddress', inet_pton($ip));
 
         $matchingUserIDs = $sql->get()->resultArray();
+        if(!empty($matchingUserIDs)) {
+            foreach ($matchingUserIDs as $matchingUserID) {
```

Comment:
I think you can simplify this using [array_column](http://php.net/array_column).